<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAeroDiagForm</class>
 <widget class="QWidget" name="CreateAeroDiagForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>74</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Аэрологическая диаграмма</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="StationWidget" name="stWidget">
     <property name="isHorisontal" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="typeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>По станциии</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По анализу</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По спутнику</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Срок:</string>
       </property>
       <property name="buddy">
        <cstring>departureTime</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="departureTime">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="departureDate">
       <property name="displayFormat">
        <string>&quot;dd&quot; MMMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCenter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Центр:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="spaceCenterCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelForecTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Срок прогноза:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="forecTimeSpin"/>
     </item>
     <item>
      <widget class="QPushButton" name="runBtn">
       <property name="text">
        <string>Сформировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="restoreDataCheck">
       <property name="text">
        <string>Восстановление данных</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>departureTime</tabstop>
  <tabstop>departureDate</tabstop>
  <tabstop>spaceCenterCombo</tabstop>
  <tabstop>forecTimeSpin</tabstop>
  <tabstop>runBtn</tabstop>
  <tabstop>restoreDataCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
